### PR TITLE
Add description for Sleep Timer services of Yamaha MusicCast

### PR DESCRIPTION
**Pull request by micha91**
## Proposed change

In this PR we add the documentation of the sleep timer related services, introduced in the referenced PR.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/57910
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/yamaha_musiccast.markdown [hunk 1]
```diff
@@ -57,6 +57,35 @@ data:
   media_content_id: "presets:1"
   media_content_type: "music"
 ```
+
+## Sleep Timer services
+
+Some MusicCast devices support sleep timers. Sleep timers can be used to turn of the device after a given amount of minutes.
```
Comment (frenck) [inline source/_integrations/yamaha_musiccast.markdown]:
```suggestion
Some MusicCast devices support sleep timers. Sleep timers can be used to turn off the device after a given amount of minutes. 
```

File: source/_integrations/yamaha_musiccast.markdown [hunk 2]
```diff
@@ -57,6 +57,35 @@ data:
...
+The currently set sleep time is also available as state attribute `sleep_time` for sleep timer supporting devices. If the sleep timer is disabled, the value of `sleep_time` is set to 0.
+
+### Examples
+
+Set a sleep timer to 90 minutes:
+
+```yaml
+service: yamaha_musiccast.set_sleep_timer
+target:
+  entity_id: media_player.wohnzimmer_zone2
```
Comment (frenck) [inline source/_integrations/yamaha_musiccast.markdown]:
```suggestion
  entity_id: media_player.living_room_zone2
```

File: source/_integrations/yamaha_musiccast.markdown [hunk 3]
```diff
@@ -57,6 +57,35 @@ data:
...
+data:
+  sleep_time: 90
+```
+
+Clear a sleep timer/disable it:
+
+```yaml
+service: yamaha_musiccast.clear_sleep_timer
+target:
+  entity_id: media_player.wohnzimmer_zone2
```
Comment (frenck) [inline source/_integrations/yamaha_musiccast.markdown]:
```suggestion
  entity_id: media_player.living_room_zone2
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by micha91** [comment]
will be available as select entity, see [here](https://github.com/home-assistant/core/pull/60645)

**Pull request closed without merging**